src/Events/EventDispatchableInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace League\OAuth2\Server\Events;
+
+use Psr\EventDispatcher\EventDispatcherInterface;
+
+interface EventDispatchableInterface
```

Comment:
```suggestion
interface EventDispatcherAwareInterface
```

This `SomethingAwareInterface` + `Trait` is a common naming scheme that has been used by the previous implementation as well. I don't think that we should invent something new here.

Reply:
Ok. I changed interface name.